docs(content-releases): add frontend docs (#19391)
strapi/strapi: docs/docs/docs/01-core/content-releases/02-frontend/02-details-page.mdx

Reviewer: I would name this file `02-release-details-page` since it is the name of the page in the codebase

---

> title: Release details view

Reviewer: ```suggestion
title: Release details page
```

tags:
  - content-releases
  - tech design
---

### Overview:

> The details-view offers a comprehensive presentation of all entries linked to the release. The entries are grouped into sections of tables displaying useful information about each entry such as its locale, the action to take (publish/unpublish), the status of the entry (published, unpublished, or validation errors).

Reviewer: ```suggestion
The release details page allows users with appropriate permissions to do the following:
```

> On this page, users with the appropriate permissions can do the following:

Reviewer: ```suggestion

```

#### Group entries:

> Entries associated with a release can be systematically organized in tables using various grouping options:
> 
> - <strong>by Content-Types</strong>: Organize entries by category.
> - <strong>by Locales</strong>: Group entries based on locale.
> - <strong>by Actions</strong>: Categorize entries by action to perform, making it easier to discern
>   between entries to publish and entries to unpublish.

Reviewer: ```suggestion
Entries in a release can be be grouped by the following options:

- Content-Types: Group all entries with the same content type together
- Locales: Group all entries with the same locale together.
- Actions: Group all entries with the same action (publish/unpublish) together.
```

#### Edit a release:

If the user has the `plugin::content-releases.update` permissions, a "three dot" button will appear in the header giving access to a menu with an "Edit" option.

> The Edit button opens a modal, enabling users to modify the release name. To save the changes, the new name must be unique, non-empty, and distinctly modified.

Reviewer: The uniqueness only applies to pending releases

#### Delete a release:

> If the user has the `plugin::content-releases.delete` permissions, a "three dot" button will appear in the header giving access to a menu with a "Delete" option.

Reviewer: ```suggestion
If the user has the `plugin::content-releases.delete` permissions, a "three dot" button will appear in the header giving access to a menu with a "Delete" option. Selecting this option triggers a confirmation modal.
```

> Upon activation, the "Delete" button triggers a confirmation modal, prompting the user to confirm the deletion.

Reviewer: ```suggestion

```

#### View Entry Status

> Each entry within a release is assigned a status, indicating its readiness for actions such as Publish/Unpublish or any validation errors present. If the entry has a validation errors. A user can use the entry's "Three dots" menu to access a "Edit entry" link to the CM edit view.

Reviewer: ```suggestion
Each entry within a release is assigned a status, indicating its readiness for actions such as Publish/Unpublish or any validation errors present. If the entry has validation errors, a user can use the entry's "three dots" menu to access the "Edit entry" link which navigates directly to the entry in the content manager.
```
